lib/v2/bsky/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/keyword/:keyword/:routeParams?': ['untitaker'],
```

Comment:
https://github.com/DIYgod/RSSHub/blob/559c12fade224864bdb4833fbc63b7cd59e644f9/lib/v2/bsky/router.js#L2

```suggestion
    '/keyword/:keyword': ['untitaker'],
```